README.md

```diff
@@ -1,6 +1,41 @@
 # helm-operator
 
-[![Build Status](https://github.com/joelanford/helm-operator/workflows/CI/badge.svg?branch=master)
-[![Coverage Status](https://coveralls.io/repos/github/joelanford/helm-operator/badge.svg?branch=master)](https://coveralls.io/github/joelanford/helm-operator?branch=master)
+[![Build Status](https://github.com/stackrox/helm-operator/workflows/CI/badge.svg?branch=main)
 
 Experimental refactoring of the operator-framework's helm operator
+
+### Why a fork?
+
+As the helm-operator is an experimental refactoring and not actively maintained we started a fork to 
+further support [hybrid operators](https://github.com/operator-framework/operator-sdk/issues/670) based on Helm.
+
+This fork should used as a library and not is recommended to watch CustomResources by configured `watches`.
+
+### Quickstart
+
+Add this lib as a replace directive to your `go.mod`:
+
+```
+replace(
```

Comment:
super nit: it's common to just provide a CLI command, e.g., `go mod edit -replace=github.com/joelanford/helm-operator=github.com/stackrox/helm-operator@main` (also, the line as it's written is invalid, as you need to specify a revision for the replacement module)

Reply:
done